core-bundle/src/EventListener/DbafsMetadataSubscriber.php

```diff
@@ -114,7 +119,7 @@ private function getDefaultLocales(): array
             return $this->defaultLocales;
         }
 
-        $page = $GLOBALS['objPage'] ?? null;
+        $page = $this->requestStack->getMainRequest()?->attributes?->get('pageModel');
```

Comment:
```suggestion
        $page = $this->requestStack->getMainRequest()?->attributes->get('pageModel');
```

If there is a request object, it will have an `attributes` property.